NASA-AMMOS/aerie-ui
# feat: batch delete activities with ag-grid

This resolves https://jira.jpl.nasa.gov/browse/AERIE-1944

To test:

1. Create a plan with multiple activities
2. Click an activity in the activity table
3. Shift (or cmd/ctrl) + click another activity to select multiple rows
4. Right click to bring up context menu
5. Click "Delete # Activities"
6. Verify that the selected activities are deleted
